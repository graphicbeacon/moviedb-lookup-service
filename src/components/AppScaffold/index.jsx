import React from 'react';

const AppScaffold = () => (
  <>
    <nav className="navbar navbar-expand-lg navbar-dark bg-dark">
      <a className="navbar-brand" href="#">
        TheMovieDB Lookup Service
      </a>
    </nav>
    <div className="container">{/* TODO Implement SearchProvider */}</div>
  </>
);

export default AppScaffold;
