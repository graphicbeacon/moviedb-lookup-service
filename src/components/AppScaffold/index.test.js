import React from 'react';
import TestRenderer from 'react-test-renderer';
import AppScaffold from './index';

describe('<App />', () => {
  test('should render without throwing', () => {
    const app = TestRenderer.create(<AppScaffold />);
    const appInstance = app.root;
    expect(appInstance.type).toBe(AppScaffold);
  });
});
