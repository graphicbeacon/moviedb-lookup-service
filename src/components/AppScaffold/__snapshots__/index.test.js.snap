// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppScaffold /> should match generated snapshot 1`] = `
Array [
  <nav
    className="navbar navbar-expand-lg navbar-dark bg-dark"
  >
    <a
      className="navbar-brand"
      href="#"
    >
      TheMovieDB Lookup Service
    </a>
  </nav>,
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col-sm"
      >
        <form
          className="mt-5 mb-5"
          data-testid="searchForm"
          onSubmit={[Function]}
        >
          <div
            className="input-group"
          >
            <input
              className="form-control form-control-lg"
              data-testid="searchInput"
              onChange={[Function]}
              type="search"
              value=""
            />
            <div
              className="input-group-append"
            >
              <button
                className="btn btn-primary"
                data-testid="searchSubmit"
                type="submit"
              >
                Search
              </button>
            </div>
          </div>
        </form>
      </div>
    </div>
  </div>,
]
`;
